test/WebJobs.Script.Tests.Integration/Controllers/ControllerScenarioTestFixture.cs

```diff
@@ -48,10 +60,11 @@ public ControllerScenarioTestFixture(bool isAuthDisabled)
 
         public HttpClient HttpClient { get; set; }
 
-        public HttpServer HttpServer { get; set; }
+        public TestServer HttpServer { get; set; }
 
-        protected virtual void RegisterDependencies(ContainerBuilder builder, WebHostSettings settings)
+        protected virtual void ConfigureWebHostBuilder(IWebHostBuilder webHostBuilder)
         {
+            webHostBuilder.ConfigureServices(c => c.AddSingleton(HostSettings));
```

Comment:
This hook enables deriving fixtures to register services used by the test server.